src/Umbraco.Web.UI.Client/src/packages/core/entity-action/entity-action.event.ts

```diff
@@ -2,7 +2,9 @@ import { UmbControllerEvent } from '@umbraco-cms/backoffice/controller-api';
 import type { UmbEntityModel } from '@umbraco-cms/backoffice/entity';
 
 // eslint-disable-next-line @typescript-eslint/no-empty-object-type
-export interface UmbEntityActionEventArgs extends UmbEntityModel {}
+export interface UmbEntityActionEventArgs extends UmbEntityModel {
+	discriminator?: string;
```

Comment:
a discriminator should ideally/conceptually be a method.. so maybe go with the name `originUnique` or something in tht line.